evaluator/evaluator/app/core/task_helper.py

```diff
@@ -0,0 +1,76 @@
+import logging
+
+import requests
+from evaluate import list_evaluation_modules
+
+from evaluator.app.models import get_dataset_metadata
+
+logger = logging.getLogger(__name__)
+
+
+def task_id(task_name, skill_id, dataset_name, metric_name=None) -> str:
+    """
+    Generates a task id.
+
+    Args:
+        task_name (str): Name of the task.
+        skill_id (str): ID of the skill.
+        dataset_name (str): Name of the dataset.
+        metric_name (str): Optional name of a metric.
+
+    Returns: Task id.
+    """
+    task_id = f"{task_name}-{skill_id}-{dataset_name}"
+    if metric_name is not None:
+        task_id = f"{task_id}-{metric_name}"
+    return task_id
+
+
+def skill_exists(skill_id, token) -> bool:
+    """
+    Checks if the given skill exists.
+
+    Args:
+        skill_id (str): ID of the skill.
+        token (str): Authentication token. Required for non-public skills.
+
+    Returns: True if the skill exists. Otherwise False.
+    """
+    headers = {}
+    if token:
+        headers = {"Authorization": f"Bearer {token}"}
+
+    response = requests.get(
+        f"https://square.ukp-lab.de/api/skill-manager/skill/{skill_id}", headers=headers
```

Comment:
Can you use an env variable for the host (https://square.ukp-lab.de/api) I think we use `SQUARE_API_URL`. This way, running it on a different domain will be easily configurable.

Reply:
Done ✅